doc/source/installation.rst

```diff
@@ -2,5 +2,211 @@
+
+After installation, add an environment variable to your ``.bashrc`` either manually or through the following command:
+
+.. code-block:: text
+
+   echo "export DEAL_II_DIR=/home/username/deal.ii-candi/deal.II-<version>" >> ~/.bashrc`
+
+Installing deal.II manually (Step #1)
+--------------------------------------
+.. note:: 
+  If you have installed deal.II through candi, you can skip right away to :ref:`install-lethe`
+
+We first need to install the deal.II dependencies.
+
+
+MPI
+~~~~~
+
+MPI, for Message Passing Interface, is required for the installation of all components of Lethe. Therefore it should be the first thing you install. On Debian Linux based OS, MPI can be installed directly from your package manager. 
+
+As an example, in Ubuntu:
+
+.. code-block:: text
+
+  sudo apt-get install libopenmpi-dev openmpi-bin openmpi-common
+
+In Manjaro or other arch based linux distribution:
+
+.. code-block:: text
+  
+  sudo pacman -Sy openmpi
+
+
+numdiff
+~~~~~~~~
+
+numdiff is used within the automatic testing procedure of Lethe to compare files obtained through floating point arithmetic. Without numdiff, Lethe automatic tests may fail when they should not. Numdiff can be installed directly from your package manager.
+
+.. code-block:: text
+
+  sudo apt-get install numdiff
+
+Regrettably, numdiff is not available in the pacman package manager. It can be downloaded from the following `website <http://www.nongnu.org/numdiff/>`_. If you are using an arch distribution, we assume that you will already know how to carry on with the installation of numdiff. 
+
+P4est
+~~~~~~~
+
+To install p4est, the usual `installation procedure <https://www.dealii.org/current/external-libs/p4est.html>`_ of deal.II can be followed.
+
+
+
+Trilinos
+~~~~~~~~~
+
+The installation of Trilinos should be done using the `installation procedure <https://www.dealii.org/current/external-libs/trilinos.html>`_ of deal.II.
+
+
+
+METIS
+~~~~~~~
+
+METIS is used for mesh partitioning for parallel computing purposes, specifically in cases with simplex grids. It can be downloaded `here <http://glaros.dtc.umn.edu/gkhome/metis/metis/download>`_ or through candi.
+
+
+
+Installation of deal.II
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Clone deal.II from the `official repository <https://github.com/dealii/dealii>`_
+
+.. code-block:: text
+
+  git clone https://github.com/dealii/dealii 
+
+Configure deal.II in a build folder at the same level as the source code
+
+.. code-block:: text
+
+  mkdir build
+  cd build
+
+Depending on how you have installed p4est, Trilinos and METIS, you may need to specify the installation folder of the three libraries
+
+.. code-block:: text
+
+  cmake ../dealii -DDEAL_II_WITH_MPI=ON -DDEAL_II_WITH_TRILINOS=ON -DTRILINOS_DIR=path/to/your/trilinos/installation -DDEAL_II_WITH_P4EST=ON -DP4EST_DIR=path/to/your/p4est/installation  -DDEAL_II_WITH_METIS=ON -DMETIS_DIR=path/to/your/metis/installation -DCMAKE_INSTALL_PREFIX=/path/to/desired/installation`
```

Comment:
Should we talk about ccmake? I always use this now and it feels easier

Reply:
I'm not sure i'd go there here for now. It's a good idea for a later addition, but then the installation become CMAKE instructions and etc. I'd rather keep it in the most simple format as possible.

Reply:
That makes sense